### PR TITLE
[release-0.6] docs: document rdt related annotations

### PR DESCRIPTION
**Pull request by marquiz**
(cherry picked from commit 02a79520594b77d85d9c96bf7d65a405839762d5)

### CODE REVIEW

File: docs/policy/rdt.md [hunk 1]
```diff
@@ -24,15 +24,30 @@ Classes of Service (CLOS) (under `/sys/fs/resctrl`) corresponding the RDT class.
...
+
+The default behavior can be overridden with pod annoations:
+
+- `rdtclass.cri-resource-manager.intel.com/pod: <class-name>` specifies a
+  pod-level default that will be used for all containers of a pod
+- `rdtclass.cri-resource-manager.intel.com/container.<container-name>: <class-name>`
+   specifies container-specific assignment, taking preference over possible
+   pod-level annotation (above)
+
+With pod annoations it is possible to specify RDT classes other than
```
Comment (jukkar) [inline docs/policy/rdt.md:46]: typo `s/annoations/annotations`

Comment (marquiz) [reply docs/policy/rdt.md:46]: Woops, thanks for spotting this. This is a backport, though, so -> #776

Comment (marquiz) [reply docs/policy/rdt.md]: #776 cherry-picked

**Comment by marquiz** [comment]
ping @jukkar

**Review by jukkar — APPROVED** [review]

**Pull request merged**